apps/lido/arapp.json

```diff
@@ -42,6 +39,11 @@
       "name": "Deposit role",
       "id": "DEPOSIT_ROLE",
       "params": []
+    },
+    {
+      "name": "Set MEV tx fee withdrawal limit role",
+      "id": "SET_MEV_TX_FEE_WITHDRAWAL_LIMIT_ROLE",
```

Comment:
We also need to update the role names, ie
`SET_MEV_TX_FEE_WITHDRAWAL_LIMIT_ROLE ` --> `SET_EL_REWARDS_WITHDRAWAL_LIMIT_ROLE`

(and update the "name" field above)

Reply:
Done